src/pages/signup-affiliates/utils/_utils.tsx

```diff
@@ -69,3 +69,22 @@ export const Submit = ({
             }),
         })
 }
+
+export const trackEvent = ({
+    action,
+    email,
```

Comment:
does email need to be added to the type and specify type for it ? 

<img width="864" alt="Screenshot 2024-01-03 at 2 59 35 PM" src="https://github.com/binary-com/deriv-com/assets/64970259/3dfc5319-50d6-4101-8554-5e098beb98a7">

Reply:
yes @mitra-deriv that should be added to types
I'll do it once `@deriv/analytics` will be updated